tests/evaluation/neg8/neg8.dl

```diff
@@ -0,0 +1,17 @@
+.decl b(x: number)
```

Comment:
Add "souffle devs 2020" header and a very short blurb explaining what behaviour this test case exercises.

Reply:
added header